### ISSUE TITLE
Shortcut to quickly close and switch to other databases

### ISSUE THREAD

**Issue by Context6016** [Kunzisoft/KeePassDX#1924]
I'm someone that uses multiple databases regularly.
Is it possible to add a floating button shortcut (similiar to the "lock database" icon) to quickly close the current database and go back to the database selection screen?

Currently, when I have a database open, I have to click the back button 3 times to get to the database selection screen (first press to lock current database, 2nd press to cancel the keyboard password entry, 3rd press to finally go back to the home screen)

Alternatively, is it possible to add a database as a shortcut to the home screen? Or it could be a long press context menu on the app icon itself.

**Comment by pscheit** [comment]
I agree it's a bit tricky to access more than one database at a time. And I dont think it supports AutoOpen? (KeeAutoExec)

**Comment by deutrino** [comment]
I also use one database for passwords and another for TOTP credentials, and this would be a big improvement.

**Comment by deutrino** [comment]
See also #209.
